current/link-analysis.md

```diff
@@ -11,7 +19,9 @@ Most of the following examples show the **domain** to **domain** links. For exam
 
 We do provide one example below that provides raw data, however.
 
-### Extraction of Simple Site Link Structure
+## Extraction of Simple Site Link Structure
```

Comment:
Let's rephrase everything so the headings can be of the form "How do I..."
So, something like "Extract Simple Site Link Structure"?